### PR TITLE
new quiz required json format

### PR DESCRIPTION
**Pull request by IwoSzczepaniak**
I based the JSON format on the assumption that it is easier for the teacher to provide the full path to the files, i.e. make a flat list of files rather than create a hierarchical structure. We can create the hierarchy on the server side after receiving the request.

### CODE REVIEW

**Review by rcybulski1122012 — APPROVED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -58,6 +58,32 @@ make task
 - for every 'extra'(not correct) smell -> -1 point.
 - no negative points
 
+## Required JSON format to add new quiz
+```json
+{
+    "smell_types": ["Unnecessary Comment", "Logic Error", "Incorect Typing"],
+    "files": [
+        {
+            "file_id": "path",
```
Comment (Fifus17) [inline README.md]: Wouldn't file_path instead of file_id bring more information what should be passed in request?

Comment (BARBUR02) [reply README.md]: Agree

Comment (IwoSzczepaniak) [reply README.md]: Yeah, sure

**Pull request merged**